### PR TITLE
Add pbnj

### PR DESCRIPTION
**Pull request by mmlb**
Depends on #65.

### CODE REVIEW

File: content/components/_index.md [hunk 1]
```diff
@@ -12,21 +12,25 @@ Handles DHCP requests, hands out IPs, and serves up iPXE.
...
+It collects data from Tinkerbell and transforms it into a JSON format to be consumed as metadata.
+
+### OSIE
 
 Installs operating systems and handles deprovisioning.
 
-### Tinkerbell
+### PBnJ
+
+Communicates with BMCs to ccontrol power and boot settings.
```
Comment (grahamc) [inline content/components/_index.md:26]:
```suggestion
Communicates with BMCs to control power and boot settings.
```

Comment (mmlb) [reply content/components/_index.md]: fixed, thanks for spotting it.

File: content/homepage/section-1.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ hidden = true
 +++
 
 Tinkerbell is a bare metal provisioning engine, built and maintained by the team at <a href="https://www.packet.com/">Packet</a>.
-Tinkerbell consists of four microservices: boots, OSIE, tink, and hegel.
+Tinkerbell consists of four microservices: Boots, Hegel, OSIE, PBnJ, and Tink.
```
Comment (kdeng3849) [inline content/homepage/section-1.md:9]: Shouldn't it be five here?

Comment (mmlb) [reply content/homepage/section-1.md]: Yes, thanks. I noticed this during one of the rebases but forgot to fix, 1s.

Comment (mmlb) [reply content/homepage/section-1.md]: Good catch. I noticed during one of the rebases but forget to go back and fix. I also went looking for other instances of "[fF]our" and found other places to fix.

**Comment by mmlb** [comment]
@ronggur I don't know what to do / how to fix 
image

**Comment by ronggur** [comment]
@mmlb sorry, would you tell me what need to be fix?

**Comment by kdeng3849** [comment]
@ronggur I think he's talking about how the PBnJ box is sticking out on a separate line

**Comment by ronggur** [comment]
@felixwidjaja any idea the best way to handle this? feel too tight?

Screen Shot 2020-06-02 at 11 50 30

**Comment by ronggur** [comment]
@mmlb @kdeng3849 do you want to merge it first and then I'll create another PR to fix it? or what is the best way?

**Comment by mmlb** [comment]
@ronggur you can make the commit directly to this pr and I'll squash it into my commit and then we can merge it.

**Pull request merged**